west.yml

```diff
@@ -35,6 +35,8 @@ manifest:
       url-base: https://github.com/PelionIoT
     - name: memfault
       url-base: https://github.com/memfault
+    - name: vge0rge_remote
```

Comment:
When pulling in a PR we don't add a new remote. Instead change we set the revision to `pull/181/head` where 181 is the PR number as in https://github.com/nrfconnect/sdk-mcuboot/pull/181 <--

Reply:
Forgot to update indeed, this is fixed now!